codes/kotlin/chapter_backtracking/permutations_i.kt

```diff
@@ -0,0 +1,57 @@
+/**
+ * File: permutations_i.kt
+ * Created Time: 2024-01-25
+ * Author: curtishd (1023632660@qq.com)
+ */
+
+package chapter_backtracking
```

Comment:
Can the package be the file name? We don't have to consider the naming collision problem in this situation and the code will be cleaner.

Reply:
alright

Reply:
I don't quite understand what you mean, do you mean simply deleting the entire package name and wrapping the method into classes ?

Reply:
You add `class permutations_i` to avoid function naming collision because Kotlin doesn't allow the functions to share the same names + parameters in a package.

Therefore, we can set different package names for different files to avoid the naming collision problem. Then we can remove `class permutations_i`.

Reply:
Alright, I understand.